src/wombats/daos/game.clj

```diff
@@ -105,7 +112,7 @@
     (let [frame-tmp-id (d/tempid :db.part/user)
           frame-trx {:db/id frame-tmp-id
                      :frame/frame-number 0
-                     :frame/round-number 0
+                     :frame/round-number 1
```

Comment:
So I don't think there is. Frame 0 does exist, it's the state of the arena prior to the start of the game. Round 0 on the other hand does not exist.